src/Traversable.ts

```diff
@@ -22,131 +22,215 @@ import {
   getFunctorComposition
 } from './Functor'
 import { HKT, Type, Type2, Type3, URIS, URIS2, URIS3 } from './HKT'
-import { Witherable, Witherable1, Witherable2, Witherable2C, Witherable3, Witherable3C } from './Witherable'
-import { some } from './Option'
 
-/** @typeclass */
+/**
+ * Interface for {@link Traversable.traverse}
+ * @since 1.7.0
+ */
+export interface Traverse<T> {
+  <F extends URIS3, FU, FL>(F: Applicative3C<F, FU, FL>): <A, B>(
```

Comment:
IME 3C must go after 3, and 2C after 2 otherwise typescript might pick the wrong overloading

Reply:
I've never faced problems with the order, but ok I'll change

Reply:
Alas I did, that's why all overloadings in fp-ts follow the pattern

- 3
- 3C
- 2
- 2c
- 1